com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -678,6 +934,11 @@ internal void SerializeSceneObject(NetworkWriter writer, ulong targetClientId)
                 writer.WriteBool(false);
             }
 
+            {
+                var (isReparented, latestParent) = GetNetworkParenting();
+                WriteNetworkParenting(writer, isReparented, latestParent);
```

Comment:
Should this be? 
```suggestion
                writer.WriteUInt64Packed(NetworkObjectId);
                WriteNetworkParenting(writer, isReparented, latestParent);
```

That is, when this write function was invoked earlier it had what I assume to be the id of the object whose parenting is changing.  Here we don't do that, and it's not clear to me how the recipient can tell what object ID this goes with

Reply:
> how the recipient can tell what object ID this goes with.

method encapsulates this code block starts as:

```cs
internal void SerializeSceneObject(NetworkWriter writer, ulong targetClientId)
{
        writer.WriteBool(IsPlayerObject);
        writer.WriteUInt64Packed(NetworkObjectId);
        writer.WriteUInt64Packed(OwnerClientId);
        // ...
```

and this is a `NetworkObject` serialization for objects that existed in the scene, the recipient will know about them for sure, we're just adding a few extra values to this `NetworkObject` serialization payload (and read it back on the other side in `DeserializeSceneObject()` method.